django_freeradius/tests/utils.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from django.core.management import call_command
+
+
+class FileMixin(object):
+    def _get_path(self, file):
+        d = os.path.dirname(os.path.abspath(__file__))
+        return os.path.join(d, file)
+
+
+class CallCommandMixin(object):
+    def _call_command(self, command, **kwargs):
+        call_command(command, **kwargs)
```

Comment:
why not move all the test mixins in the same file? Either here or in `__init__.py`? Wouldn't it make it easier to find these mixins when reading the source code?